src/scmdata/time.py

```diff
@@ -137,14 +137,15 @@ def to_index(self) -> pd.Index:
 
     def as_cftime(self) -> list:
```

Comment:
Perhaps rather than hard coding, add a `date_cls` parameter which defaults to `cftime.DatetimeGregorian`. While not applicable to this use-case i think that it may come in handy if dealing with 360 day calendars

Reply:
Ye nice, done